core/tests/protractor_desktop/additionalEditorFeatures.js

```diff
@@ -275,8 +275,11 @@ describe('Full exploration editor', function() {
       await responseEditor.setDestination('final card', true, null);
 
       // Now, set multiple rules to a single answer group.
-      responseEditor = await explorationEditorMainTab.getResponseEditor(0);
-      await responseEditor.addRule('TextInput', 'Contains', ['meh', 'okay']);
+      await explorationEditorMainTab.moveToState('final card');
+      await explorationEditorMainTab.moveToState('first card');
```

Comment:
Can you please explain why these were added?

Reply:
https://user-images.githubusercontent.com/80846488/170272265-b203bc2e-ae43-4074-a847-03de875ce91e.mp4

You can see at 2:13, top response is not  clickable, because it become out of frame due to some css changes, so i just shuffle back to current state to see first(top) response. 

I'm not sure about this solution.

@SAEb-ai PTAL.

Reply:
Can you please add a comment explaining this?